### ISSUE TITLE
403 for apex-plugin.com ?!?

### ISSUE THREAD

**Issue by bobromeo** [Dani3lSun/awesome-orclapex#9]
Forbidden
You don't have permission to access this resource.Server unable to read htaccess file, denying access to be safe

Additionally, a 403 Forbidden error was encountered while trying to use an ErrorDocument to handle the request.

**Comment by Dani3lSun** [comment]
Yeah I think this page is down since some time...Will remove it from the list...
For finding open source plugins apex.world would be the best bet
